<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureWgt</class>
 <widget class="QMainWindow" name="CaptureWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video capture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="view">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="capture"/>
   <addaction name="separator"/>
   <addaction name="device"/>
   <addaction name="input"/>
   <addaction name="resolution"/>
   <addaction name="settings"/>
   <addaction name="cross"/>
   <addaction name="laser"/>
   <addaction name="flipX"/>
   <addaction name="flipY"/>
   <addaction name="separator"/>
   <addaction name="brightnessRange"/>
   <addaction name="fullContrast"/>
   <addaction name="surface"/>
   <addaction name="equalizeHist"/>
   <addaction name="sobel"/>
   <addaction name="median"/>
   <addaction name="timeSmooth"/>
   <addaction name="highPass"/>
   <addaction name="separator"/>
   <addaction name="pixmap"/>
  </widget>
  <widget class="QDockWidget" name="brightnessDw">
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>128</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>48</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="capture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/arrow.png</normaloff>:/images/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </action>
  <action name="device">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/device.png</normaloff>:/images/device.png</iconset>
   </property>
   <property name="text">
    <string>Device</string>
   </property>
  </action>
  <action name="settings">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/wrench.png</normaloff>:/images/wrench.png</iconset>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="format">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/format.png</normaloff>:/images/format.png</iconset>
   </property>
   <property name="text">
    <string>format</string>
   </property>
  </action>
  <action name="align">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
   </property>
   <property name="text">
    <string>align</string>
   </property>
  </action>
  <action name="cross">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/laser.png</normaloff>:/images/laser.png</iconset>
   </property>
   <property name="text">
    <string>cross</string>
   </property>
   <property name="toolTip">
    <string>Feature emphasis</string>
   </property>
  </action>
  <action name="filters">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/filters.png</normaloff>:/images/filters.png</iconset>
   </property>
   <property name="text">
    <string>filters</string>
   </property>
   <property name="toolTip">
    <string>Image filters</string>
   </property>
  </action>
  <action name="input">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/s-video.png</normaloff>:/images/s-video.png</iconset>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
   <property name="toolTip">
    <string>Choose hardware input channel (only usable if more then one).</string>
   </property>
  </action>
  <action name="equalizeHist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/equalizeHist.png</normaloff>:/images/equalizeHist.png</iconset>
   </property>
   <property name="text">
    <string>Equalize histogram</string>
   </property>
  </action>
  <action name="fullContrast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/fullContrast.png</normaloff>:/images/fullContrast.png</iconset>
   </property>
   <property name="text">
    <string>Full contrast</string>
   </property>
  </action>
  <action name="sobel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/sobel.png</normaloff>:/images/sobel.png</iconset>
   </property>
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="median">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/median.png</normaloff>:/images/median.png</iconset>
   </property>
   <property name="text">
    <string>Median</string>
   </property>
  </action>
  <action name="timeSmooth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/timeSmooth.png</normaloff>:/images/timeSmooth.png</iconset>
   </property>
   <property name="text">
    <string>Time smoothing</string>
   </property>
  </action>
  <action name="pixmap">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/pixmap.png</normaloff>:/images/pixmap.png</iconset>
   </property>
   <property name="text">
    <string>pixmap</string>
   </property>
   <property name="toolTip">
    <string>Save current image as pixmap</string>
   </property>
  </action>
  <action name="highPass">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/highpass.png</normaloff>:/images/highpass.png</iconset>
   </property>
   <property name="text">
    <string>High pass</string>
   </property>
   <property name="toolTip">
    <string>Highpass filter</string>
   </property>
  </action>
  <action name="surface">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/surface.png</normaloff>:/images/surface.png</iconset>
   </property>
   <property name="text">
    <string>Substract surface</string>
   </property>
   <property name="toolTip">
    <string>Substract surface</string>
   </property>
  </action>
  <action name="brightnessRange">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/brightness.png</normaloff>:/images/brightness.png</iconset>
   </property>
   <property name="text">
    <string>Brightness range</string>
   </property>
   <property name="toolTip">
    <string>Brightness range</string>
   </property>
  </action>
  <action name="flipX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/flipX.png</normaloff>:/images/flipX.png</iconset>
   </property>
   <property name="text">
    <string>Flip X</string>
   </property>
   <property name="toolTip">
    <string>Flip an image around X axis</string>
   </property>
  </action>
  <action name="flipY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/flipY.png</normaloff>:/images/flipY.png</iconset>
   </property>
   <property name="text">
    <string>Flip Y</string>
   </property>
   <property name="toolTip">
    <string>Flip an image around Y axis</string>
   </property>
  </action>
  <action name="resolution">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/resolution.png</normaloff>:/images/resolution.png</iconset>
   </property>
   <property name="text">
    <string>Video resolution</string>
   </property>
   <property name="toolTip">
    <string>Preferred video resolutions list</string>
   </property>
  </action>
  <action name="laser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/laser02.png</normaloff>:/images/laser02.png</iconset>
   </property>
   <property name="text">
    <string>laser</string>
   </property>
   <property name="toolTip">
    <string>Laser spot position</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
